src/python/tests/test_configuration_manager.py

```diff
@@ -0,0 +1,653 @@
+                                      'default': 'False'}})
+
+        await create_category(category_name='boolean',
+                              category_description='boolean type',
+                              category_value={'data': {
+                                  'description': 'boolean type with default True',
+                                  'type': 'boolean',
+                                  'default': 'True'}},
+                              keep_original_items=True)
+        try:
+            async with aiopg.sa.create_engine(_CONNECTION_STRING) as engine:
+                async with engine.acquire() as conn:
+                    async for result in conn.execute(select_value_stmt):
+                        assert sorted(list(result[0].keys())) == ['data', 'info']
+                        # Test 'data' category
+                        assert result[0]['data']['description'] == (
+                            'boolean type with default True')
+                        assert result[0]['data']['type'] == 'boolean'
+                        assert result[0]['data']['default'] == 'True'
+                        # Test 'info' category
+                        assert result[0]['info']['description'] == (
+                            'boolean type with default False')
+                        assert result[0]['info']['type'] == 'boolean'
+                        assert result[0]['info']['default'] == 'False'
+        except Exception:
+            raise
+
+    async def test_create_category_keep_original_items_false(self):
+        """
+        Test the behavior of create_category when keep_original_items == False
+        :assert:
+            1. initial `info` data has been added
+            2. `values` dictionary only has 'data' category
+            3. values in 'data' category are as expected
+        """
+        select_value_stmt = sa.select([_configuration_tbl.c.value]).select_from(
+            _configuration_tbl).where(_configuration_tbl.c.key == 'boolean')
+        await create_category(category_name='boolean', category_description='boolean type',
+                              category_value={'info': {
+                                  'description': 'boolean type with default False',
+                                  'type': 'boolean',
+                                  'default': 'False'}})
+        try:
+            async with aiopg.sa.create_engine(_CONNECTION_STRING) as engine:
+                async with engine.acquire() as conn:
+                    async for result in conn.execute(select_value_stmt):
+                        assert sorted(list(result[0].keys())) == ['info']
+                        assert result[0]['info']['description'] == (
+                            'boolean type with default False')
+                        assert result[0]['info']['type'] == 'boolean'
+                        assert result[0]['info']['default'] == 'False'
+        except Exception:
+            raise
```

Comment:
this try block looks redundant check to me, 

`assert sorted(list(result[0].keys())) == ['data']` and following will ensure it has been replaced!

Reply:
No harm though!